### PR TITLE
Renaming namespace moq to quicr.

### LINKED ISSUE #226: Move namespace to quicr
Move namespace from moq:: to quicr::
Remove qtransport namespace and make it quicr:: as well. Let's not  do sub-namespace for this.
update the documentation to reflect the same

### PR DESCRIPTION
**Pull request by GhostofCookie**
Closes #226

### CODE REVIEW

File: cmd/examples/client.cpp [hunk 1]
```diff
@@ -270,8 +272,8 @@ main(int argc, char* argv[])
       .set_tab_expansion()
       //.allow_unrecognised_options()
       .add_options()("h,help", "Print help")("d,debug", "Enable debugging") // a bool parameter
-      ("r,url", "Relay URL", cxxopts::value<std::string>()->default_value("moq://localhost:1234"))(
-        "e,endpoint_id", "This client endpoint ID", cxxopts::value<std::string>()->default_value("moq-client"))(
+      ("r,url", "Relay URL", cxxopts::value<std::string>()->default_value("quicr://localhost:1234"))(
+        "e,endpoint_id", "This client endpoint ID", cxxopts::value<std::string>()->default_value("quicr-client"))(
```
Comment (suhasHere) [inline cmd/examples/client.cpp:276]: relay url still needs to start with moq://

File: include/quicr/detail/messages.h [hunk 2]
```diff
@@ -39,7 +39,7 @@ namespace moq::messages {
         GOAWAY_TIMEOUT = 0x10,
     };
 
-    // Ref: https://moq-wg.github.io/moq-transport/draft-ietf-moq-transport.html#name-messages
+    // Ref: https://quicr-wg.github.io/quicr-transport/draft-ietf-quicr-transport.html#name-messages
```
Comment (suhasHere) [inline include/quicr/detail/messages.h:42]: this has to be moq

**Review by suhasHere — COMMENTED** [review]
LGTM. have few nits.

Were we able to run and test a sample chat app or date server , just to make sure the basic functionality works as expected.

File: cmd/examples/client.cpp (hunk 1 above)
Comment (GhostofCookie) [reply cmd/examples/client.cpp:276]: Good catch!

File: include/quicr/detail/messages.h (hunk 2 above)
Comment (GhostofCookie) [reply include/quicr/detail/messages.h:42]: Good catch!

**Pull request merged**